docs/libcurl/opts/CURLOPT_SSLVERSION.3

```diff
@@ -50,6 +50,18 @@ TLSv1.1 (Added in 7.34.0)
 TLSv1.2 (Added in 7.34.0)
 .IP CURL_SSLVERSION_TLSv1_3
 TLSv1.3 (Added in 7.52.0)
+.IP CURL_SSLVERSION_MAX_DEFAULT
+Use as flag with CURL_SSLVERSION_TLSv1_x and behavior is same as
+CURL_SSLVERSION_MAX_TLSv1_2.
```

Comment:
Now, CURL_SSLVERSION_MAX_DEFAULT is as separeted value and every library CURL_SSLVERSION_MAX_DEFAULT must serve self (NSS via SSL_VersionRangeGetDefault and others as CURL_SSLVERSION_MAX_TLSv1.2).